monitoring/uss_qualifier/scenarios/astm/netrid/common/aggregate_checks.py

```diff
@@ -61,19 +73,72 @@ def __init__(
                     break
 
     def run(self):
+
         self.begin_test_scenario()
         self.record_note("participants", str(self._participants_by_base_url))
         self.record_note("nb_queries", str(len(self._queries)))
 
+        # DP performance
         self.begin_test_case("Performance of Display Providers requests")
-
         self.begin_test_step("Performance of /display_data requests")
+
         self._dp_display_data_times_step()
+
         self.end_test_step()
+        self.end_test_case()
+
+        # SP performance
+        self.begin_test_case("Performance of Service Providers requests")
+        self.begin_test_step("Performance of /flights?view requests")
 
+        self._sp_flights_area_times_step()
+
+        self.end_test_step()
         self.end_test_case()
+
         self.end_test_scenario()
 
+    def _sp_flights_area_times_step(self):
+        pattern = re.compile(r"v2/uss/flights\?view=")
```

Comment:
This would catch only v22a queries.

A catch here is that for v19 the OpenAPI specs define `/v1/uss/flights` while for v22a it is `/uss/flights`. So `v2` should not be here in any case.
Also, nice to account for possible other parameters.

Here is a suggestion:
 
```suggestion
        pattern = re.compile(r"(/v1)?/uss/flights\?(\S)*view=")
```

Reply:
We can actually just match on `/uss/flights\?(\S)*view="` I guess